syndesis-builder-image-generator/src/main/java/io/syndesis/image/Application.java

```diff
@@ -60,18 +61,19 @@ public static void main(String[] args) {
     }
 
     @Override
+    @SuppressFBWarnings("DM_EXIT")
     public void run(ApplicationArguments args) {
         try {
-            System.out.println("To: "+to);
+            System.out.println("To: "+to); //NOPMD
             generateIntegrationProject(new File(to));
-            System.exit(0);
         } catch (IOException e) {
-            System.exit(1);
+            e.printStackTrace(); //NOPMD
+            System.exit(1); //NOPMD
         }
     }
```

Comment:
Should we reimplement this as [CommandLineRunner](https://docs.spring.io/spring-boot/docs/current/reference/html/boot-features-spring-application.html#boot-features-command-line-runner), happy to have it in another PR.

Reply:
Good idea. I will look at that soon, after finishing off the keycloak removal stuff now.